ui/intermittent-failures/DateRangePicker.jsx

```diff
@@ -26,75 +27,58 @@ export default class DateRangePicker extends React.Component {
   };
 
   showFromMonth = () => {
-    const { from } = this.state;
-    if (!from) {
+    const { startDate } = this.state;
+    if (!startDate) {
       return;
     }
-    this.to.getDayPicker().showMonth(from);
+    this.to.getDayPicker().showMonth(startDate);
   };
 
-  fromChange = from => {
-    this.setState({ from }, () => {
-      if (!this.state.to) {
+  fromChange = startDate => {
+    this.setState({ startDate }, () => {
+      if (!this.state.endDate) {
         this.focusTo();
       }
     });
   };
 
-  toChange = to => {
-    this.setState({ to }, this.showFromMonth);
+  toChange = endDate => {
+    this.setState({ endDate }, this.showFromMonth);
   };
 
   updateData = () => {
-    const { from, to } = this.state;
+    const { startDate, endDate } = this.state;
 
-    const startday = ISODate(moment(from));
-    const endday = ISODate(moment(to));
+    const startday = ISODate(moment(startDate));
+    const endday = ISODate(moment(endDate));
 
+    this.setState(() => ({ calendarFocused: false }));
     this.props.updateState({ startday, endday });
   };
 
+  onFocusChange = calendarFocused => {
+    this.setState(() => ({ calendarFocused }));
+  };
+
   render() {
-    const { from, to } = this.state;
-    const modifiers = { start: from, end: to };
+    const { startDate, endDate } = this.state;
 
     return (
       <div className="InputFromTo d-inline-block">
-        <DayPickerInput
-          value={from}
-          placeholder="From"
-          formatDate={formatDate}
-          parseDate={parseDate}
-          format="ddd MMM D, YYYY"
-          dayPickerProps={{
-            selectedDays: [from, { from, to }],
-            toMonth: to,
-            modifiers,
-            numberOfMonths: 2,
-          }}
-          onDayChange={this.fromChange}
+        <DatePickerAirbnb
+          startDate={startDate}
+          startDateId="startDateId"
+          endDate={endDate}
+          endDateId="endDateId"
+          onDatesChange={({ startDate, endDate }) =>
+            this.setState({ startDate, endDate })
+          }
+          focusedInput={this.state.calendarFocused}
```

Comment:
Nit: You can use destructuring assignment for `calendarFocused` on line 64 instead of including 'this.state..'.